onboard/main.cpp

```diff
@@ -32,8 +34,11 @@ enum STATES{
   WAIT
 };
 int state = STATES::WAIT;
-int currLocalJoint = 0;
+int currJoint = 0;
 volatile unsigned long int timer = 0;
+volatile boolean PIDflag = true; //flag based on timer for PID control
+unsigned long int watchdog_timer; //for live control
+unsigned int WDT_allowTime = 5; //in ms
```

Comment:
If we're controlling at 100 Hz, 5ms watchdog won't work. Since a failing watchdog will make testing difficult, please only submit this after it's tested.